src/biocommons/seqrepo/_internal/translate.py

```diff
@@ -55,14 +56,14 @@ def translate_api2db(namespace, alias):
         return [
             ("VMC", "GS_" + alias if alias else None),
         ]
-    if namespace == "ga4gh":
+    if namespace == "ga4gh" and alias is not None:
```

Comment:
Is there a reason why we don't just return an empty list at the beginning if alias is none?

Reply:
Yeah, I had a hard time with this and have gone through it again. The call stacks that end up in this method are kind of messy, but I think this is the ~way it is right now:

1) `store` and its descendant functions shouldn't ever be passing `None` for an alias because it's a non-nullable field within the DB schema

2) `find_aliases` uses `None` for all fields as a wildcard indicator. So if I call this method with `("refseq", None)`, that means I want to get back all aliases with namespace `"NCBI"` (if we return an empty list if `alias` is `None`, then this lookup becomes "get all aliases regardless of namespace").

The change here is to ensure that we don't get a TypeError on the next line if `alias` is `None`. Given the structure of the 3 lines above this condition, I'm not sure if it should look like

```
    if namespace == "ga4gh" and alias is not None:
        return [
            ("VMC", "GS_" + alias[3:]),
        ]
```

or

```
    if namespace == "ga4gh":
        return [
            ("VMC", "GS_" + alias[3:] if alias else None),
        ]
```

either way, I updated the typing on the `store_alias` side of the call stack above this method to ensure that it doesn't permit `alias` to be `None` on its end.